### PR TITLE
feat(APP-3714): Rename ODS library to gov-ui-kit

### PR DESCRIPTION
**Pull request by thekidnamedkd**
## Description

Replace all docs, tokens, strings, component names, etc.

Task: [APP-3714](https://aragonassociation.atlassian.net/browse/APP-3714)

## Type of change

-   [x] New feature (non-breaking change which adds functionality)

## Checklist:

-   [x] I have selected the correct base branch.
-   [x] I have performed a self-review of my own code.
-   [x] I have commented my code, particularly in hard-to-understand areas.
-   [x] I have made corresponding changes to the documentation.
-   [x] My changes generate no new warnings.
-   [x] Any dependent changes have been merged and published in downstream modules.
-   [x] I ran all tests with success and extended them if necessary.
-   [x] I have updated the `CHANGELOG.md` file in the root folder of the package after the [UPCOMING] title and before
        the latest version.
-   [x] I have tested my code on the test network.

[APP-3714]: https://aragonassociation.atlassian.net/browse/APP-3714?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by shan8851 — CHANGES_REQUESTED** [review]
Looks good, I can see some references still in `gukModulesProvider.tsx`

```tsx
const odsModulesContextDefaults: IGukModulesContext = {
    copy: modulesCopy,
};

const odsModulesContext = createContext<IGukModulesContext>(odsModulesContextDefaults);

export const GukModulesProvider: React.FC<IGukModulesProviderProps> = (props) => {
    const {
        queryClient = defaultQueryClient,
        wagmiConfig = defaultWagmiConfig,
        wagmiInitialState,
        coreProviderValues,
        children,
        values,
    } = props;

    const contextValues = useMemo(
        () => ({
            copy: values?.copy ?? odsModulesContextDefaults.copy,
        }),
        [values],
    );

    return (
        <odsModulesContext.Provider value={contextValues}>
            <WagmiProvider config={wagmiConfig} initialState={wagmiInitialState}>
                <QueryClientProvider client={queryClient}>
                    <GukCoreProvider values={coreProviderValues}>{children}</GukCoreProvider>
                </QueryClientProvider>
            </WagmiProvider>
        </odsModulesContext.Provider>
    );
};

export const useGukModulesContext = (): IGukModulesContext => {
    const values = useContext(odsModulesContext);

    return values;
};
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -346,7 +347,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 -   Implement `DefinitionList` core component
 -   Implement `TransactionDataListItem.Skeleton` module component
--   Implement `OdsCoreProvider`, `LinkBase` and `AvatarBase` core components to support `Link` and `Image` NextJs
+-   Implement `GukCoreProvider`, `LinkBase` and `AvatarBase` core components to support `Link` and `Image` NextJs
```
Comment (cgero-eth) [inline CHANGELOG.md:350]: I would keep these names here, on previous versions the component will still be called `OdsXYZ`. Instead we can add a new entry under `unreleased` as `Rename x component to y`.
Let's specify on the changelog that we're renaming these components and the css custom variables

File: README.md [hunk 2]
```diff
@@ -12,25 +12,30 @@
...
-## License
+## Need Help? 🆘
 
-[GPL-V3](./LICENSE)
+Stuck on something? Our community is here to help! Join our [Discord channel](https://discord.com/invite/eqQJkdp) for
+support, advice, or just to share your awesome plugin creations with fellow Aragon enthusiasts.
+
+## License 📜
+
+The Aragon Governance UI Kit is released under the AGPL v3 License.
```
Comment (cgero-eth) [inline README.md:41]: The link to the LICENSE file is now missing

File: README.md [hunk 3]
```diff
@@ -12,25 +12,30 @@
...
 
 Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
 
 Please make sure to update tests as appropriate.
 
-## License
+## Need Help? 🆘
 
-[GPL-V3](./LICENSE)
+Stuck on something? Our community is here to help! Join our [Discord channel](https://discord.com/invite/eqQJkdp) for
```
Comment (cgero-eth) [inline README.md:36]: No discord links, these can get outdated

File: src/theme/tokens/index.mdx [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { Meta } from '@storybook/blocks';
 
 # Tokens
 
-The design tokens in the `@aragon/ods` UI library serve as the foundational elements for creating consistent and
+The design tokens in the `@aragon/guk` UI library serve as the foundational elements for creating consistent and
```
Comment (cgero-eth) [inline src/theme/tokens/index.mdx:7]: gov-ui-kit

File: src/theme/tokens/index.mdx [hunk 5]
```diff
@@ -14,7 +14,7 @@ used as CSS custom properties.
 
 ### With Tailwind CSS
 
-In projects using both `@aragon/ods` and `Tailwind CSS`, design tokens are readily available as Tailwind classes. For
+In projects using both `@aragon/guk` and `Tailwind CSS`, design tokens are readily available as Tailwind classes. For
```
Comment (cgero-eth) [inline src/theme/tokens/index.mdx:17]: gov-ui-kit
also below

File: src/modules/index.mdx [hunk 6]
```diff
@@ -27,15 +27,15 @@ In order to use the ODS Modules components, make sure to:
     npm install viem wagmi @tanstack/reat-query
     ```
 
--   Wrap your application using the `<OdsModulesProvider />` React Context Provider:
+-   Wrap your application using the `<GukModulesProvider />` React Context Provider:
 
     ```tsx
-    import { OdsModulesProvider } from '@aragon/ods';
+    import { GukModulesProvider } from '@aragon/guk';
```
Comment (cgero-eth) [inline src/modules/index.mdx:33]: gov-ui-kit

File: src/core/utils/formatterUtils/formatterUtils.mdx [hunk 7]
```diff
@@ -49,7 +49,7 @@ formatted:
 Import the `formatterUtils` utility and start using it as follows:
 
 ```typescript
-import { NumberFormat, formatterUtils } from '@aragon/ods';
+import { NumberFormat, formatterUtils } from '@aragon/guk';
```
Comment (cgero-eth) [inline src/core/utils/formatterUtils/formatterUtils.mdx:52]: gov-ui-kit
Same below

File: CHANGELOG.md [hunk 8]
```diff
@@ -355,7 +359,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 -   Bump minor and patch dependencies of Github workflows
 -   Expose `onValueChange` property on `Accordion` core component
 -   Update type extensions for `Tabs.Root` core component to allow forward ref
--   Update `OdsModulesProvider` component to render the `OdsCoreProvider` context.
+-   Update `GukModulesProvider` component to render the `GukCoreProvider` context.
```
Comment (cgero-eth) [inline CHANGELOG.md:362]:
```suggestion
-   Update `OdsModulesProvider` component to render the `OdsCoreProvider` context.
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -503,7 +507,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 -   Implement animations for `Dialog` and `DialogAlert` components
 -   Implement `DaoAvatar` and `MemberAvatar` module components
--   Implement `OdsModulesProvider` for using wagmi hooks on modules components
+-   Implement `GukModulesProvider` for using wagmi hooks on modules components
```
Comment (cgero-eth) [inline CHANGELOG.md:510]:
```suggestion
-   Implement `OdsModulesProvider` for using wagmi hooks on modules components
```

File: README.md [hunk 10]
```diff
@@ -12,25 +12,25 @@
...
 
 Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
 
 Please make sure to update tests as appropriate.
 
-## License
+## License 📜
 
-[GPL-V3](./LICENSE)
+The Aragon Governance UI Kit is released under the [GPL-V3](./LICENSE).
```
Comment (cgero-eth) [inline README.md:36]:
```suggestion
The Aragon Governance UI Kit is released under the [GPL-V3 License](./LICENSE).
```

**Review by cgero-eth — APPROVED** [review]
LGTM, thanks @thekidnamedkd! 🚀

**Pull request merged**